grammar.js

```diff
@@ -84,6 +85,12 @@ module.exports = grammar({
       ',',
     ),
 
+    note: () => choice(
+      'Note:', 'NOTE:', 'Notes:',
+      'Warning:', 'WARNING:',
+      'Deprecated', 'DEPRECATED:'
```

Comment:
Also `Experimental:`

Reply:
That wasn't part of the OG syntax file.

We can consider this if we officially stop trailing vimdox in favor of our own "nvimdoc" (i.e. a more strictly specified dialect).

Reply:
I suppose we can use `Warning:` instead of `Experimental:`.